app/jobs/scheduled/poll_mailbox.rb

```diff
@@ -12,6 +12,12 @@ class PollMailbox < ::Jobs::Scheduled
     def execute(args)
       @args = args
       poll_pop3 if should_poll?
+
+      DiscoursePluginRegistry.mail_pollers.each do |poller|
+        return if !poller.enabled?
+
+        poller.poll_mailbox(method(:process_popmail))
+      end
```

Comment:
Thinking hard about this one:

I'm not sure if we want the additional `||` in `site_setting.rb:119`.

The setting description for `pop3_polling_enabled?` is 

> Poll via POP3 for email replies.

I'm thinking if `should_poll` should wrap `mail_pollers` as well.

Reply:
Hello @nattsw , thanks for the review!

My reasoning here was that `SiteSetting.pop3_polling_enabled?` explicitly refers to `pop3`, so if the user disables it, we're not 100% sure that the user also wants to disable the `mail_pollers`.

Maybe we should rename the `SiteSetting.pop3_polling_enabled?` removing the reference to `pop3` and treating it as a general "polling_enabled"?

Reply:
Ahh ok. If that's the case, I think we're all good here.

Reply:
> Ahh ok. If that's the case, I think we're all good here.

So do you think we should leave this as it is now?

Thanks!

Reply:
Yup!